deployment/roles/metron_ui/tasks/main.yml

```diff
@@ -53,4 +53,4 @@
     production: no
 
 - name: Start Metron UI
-  shell: "pm2 restart {{ metron_ui_directory }}/lib/metron-ui.js --name metron"
+  shell: "pm2 start {{ metron_ui_directory }}/lib/metron-ui.js --name metron"
```

Comment:
Yes this was intentional.  Turns out the previous commit only works on AWS so I reverted it to keep the single node vagrant deployment from breaking.  @nickwallen reopened the pull request and will submit another fix.  Sorry I should have called that out in the description.